docs/getting_started.rst

```diff
@@ -116,10 +116,18 @@ Environment variables can be set in many ways, including:
 
 There are two environment variables you should plan to set:
 
-* ``KOLIBRI_RUN_MODE`` (required)
+* ``KOLIBRI_RUN_MODE`` is **required**.
 
   This variable is sent to our `pingback server <https://github.com/learningequality/nutritionfacts>`_ (private repo), and you must set it to something besides an empty string. This allows us to filter development work out of our usage statistics. There are also some `special testing behaviors <https://github.com/learningequality/nutritionfacts/blob/b150ec9fd80cd0f02c087956fd5f16b2592f94d4/nutritionfacts/views.py#L125-L179>`_ that can be triggered for special strings, as described elsewhere in the developer docs and integration testing Gherkin scenarios.
-* ``KOLIBRI_HOME`` (optional)
+  |br|
+  For example you could add this line at the end of your ``~/.bash_profile`` file:
+
+  .. code-block:: bash
+
+    export KOLIBRI_RUN_MODE="dev"
+
+
+* ``KOLIBRI_HOME`` is optional.
```

Comment:
Do the docs mention what this variable means, and where the `KOLIBRI_HOME` folder is by default e.g. `$HOME/.kolibri`?

Reply:
Good point, I cannot find a good explanation of that variable in the dev docs, but weirdly enough, there is one in the use docs 😅 

https://kolibri.readthedocs.io/en/latest/manage/command_line.html#change-the-location-of-all-kolibri-files

I'll amend that in the fist follow-up update!